src/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+  background-color: black;
+}
+
+.wrapper {
+  width: 350px;
+  margin: 20px 0 40px 20px;
+}
+
+.block-blue {
+  border: 1px solid #e75551;
+  background-color: #308dc0;
+  padding: 10px;
+}
+
+.block-yellow {
+  border: 1px solid #e75551;
+  background-color: #ffa500;
+  padding: 10px;
+}
```

Comment:
The theme of the site may be changed and binding class names to colors is bad practice. You can think of these blocks as content, wrapper, and inner wrapper. Example: outer, inner, wrapper, content, etc.